src/main/java/com/plaid/client/response/ErrorResponse.java

```diff
@@ -18,6 +18,9 @@
   private String errorMessage;
   private ErrorType errorType;
 
+  // Error code constants
+  public static String PRODUCT_NOT_READY = "PRODUCT_NOT_READY";
```

Comment:
oh since this is only used in teh test I just meant we should put it in the test file as a constant